### DISCUSSION TITLE
引用第三方库--许可证相关

### DISCUSSION THREAD

**Discussion by notplay3** [BYJRK/DotNet-Discussions#70] [通用]
我想使用HandyControl，他们使用的是MIT开源许可，如果我使用他们的库用于商业我需要在我自己的软件里包含这个MIT许可证，还是说做声明或者其他什么操作呀？？？

**Comment by BYJRK** [comment]
是的，你要包含这个许可证，写上原始版权声明

比如 PowerToys 的开放源代码通知：https://github.com/microsoft/PowerToys/blob/main/NOTICE.md

**Reply by notplay3** [reply]
明白了，谢谢
